<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWidgetClass</class>
 <widget class="QWidget" name="ServerWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_selecter" native="true">
     <property name="minimumSize">
      <size>
       <width>291</width>
       <height>191</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QTabWidget" name="tabWidget_server">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_server">
         <attribute name="title">
          <string>Сервер</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_servername">
            <property name="text">
             <string>server_name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer_server">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="pushButton_selectserver">
            <property name="text">
             <string>Выбрать</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QProgressBar" name="progressBar_capacityserver">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>11</height>
             </size>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_onlinestatus">
            <property name="text">
             <string>server_online_status</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_playerscount">
            <property name="text">
             <string>server_count_players</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_serverdescription">
         <attribute name="title">
          <string>Описание</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="textBrowser_serverdescription"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_selecterserver">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
